### PR TITLE
fix: [PIE-3783]: jenkins connector added

### PR DESCRIPTION
**Pull request by satyam-harness**
##### Jira Links:
https://harness.atlassian.net/browse/PIE-3783

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by satyam-harness** [comment]
retrigger titlecheck

**Comment by satyam-harness** [comment]
dockerize

**Comment by satyam-harness** [comment]
retrigger eslint

**Comment by satyam-harness** [comment]
retrigger typecheck

**Comment by satyam-harness** [comment]
retrigger standards

**Comment by satyam-harness** [comment]
fix prettier

**Review by kapilkumawat86 — APPROVED** [review]

File: src/modules/35-connectors/components/CreateConnector/JenkinsConnector/CreateJenkinsConnector.module.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  .dockerProviderType {
+    font-size: var(font-size-normal) !important;
+  }
+
+  .authHeaderRow {
+    display: flex;
+    justify-content: space-between;
+    margin-top: var(--spacing-medium);
+    & > [data-tooltip-id='dockerConnectorAuthentication'] {
+      margin: 0 0 7px;
```
Comment (vkbansal) [inline src/modules/35-connectors/components/CreateConnector/JenkinsConnector/CreateJenkinsConnector.module.scss:20]: use variables

Comment (satyam-harness) [reply src/modules/35-connectors/components/CreateConnector/JenkinsConnector/CreateJenkinsConnector.module.scss]: addressed

**Review by vkbansal — DISMISSED** [review]

**Review by devinder-harness — APPROVED** [review]

**Pull request merged**